common/fixation_entry_index.go

```diff
@@ -67,8 +67,9 @@ func (fs FixationStore) GetAllEntryIndices(ctx sdk.Context) []string {
 func (fs *FixationStore) getEntryVersionsFilter(ctx sdk.Context, index string, block uint64, filter func(*types.Entry) bool) (blocks []uint64) {
 	safeIndex, err := types.SanitizeIndex(index)
 	if err != nil {
-		details := map[string]string{"index": index}
-		utils.LavaError(ctx, ctx.Logger(), "getEntryVersionsFilter", details, "invalid non-ascii entry")
+		utils.LavaFormatError("invalid non-ascii entry", err,
```

Comment:
description string is a duplicate in all of the functions, change it to give us an idea where it came from

Reply:
done